client/src/datatype.tsx

```diff
@@ -0,0 +1,11 @@
+import { setupMaster } from "cluster";
+
+// example consuming code
+interface Todo {
```

Comment:
i remember this from that typescript article :') do you end up using it anywhere?